sdk/core/azure-core-amqp/test/ut/connection_tests.cpp

```diff
@@ -29,38 +29,25 @@ TEST_F(TestConnections, SimpleConnection)
 {
   {
     // Create a connection
-    Azure::Core::Amqp::_internal::Connection connection("amqp://localhost:5672", {});
+    Azure::Core::Amqp::_internal::ConnectionOptions connectOptions;
+    connectOptions.Port = Azure::Core::Amqp::_internal::AmqpPort;
 
-    Azure::Core::Amqp::_internal::ConnectionOptions options;
-    options.SaslCredentials
-        = std::make_shared<Azure::Core::Amqp::_internal::SaslPlainConnectionStringCredential>(
-            "Endpoint=sb://testHost.net/"
-            ";SharedAccessKeyName=SomeName;SharedAccessKey=SomeKey;EntityPath=testhub");
-    EXPECT_ANY_THROW(
-        Azure::Core::Amqp::_internal::Connection connection2("amqp://localhost:5672", options));
+    Azure::Core::Amqp::_internal::Connection connection("localhost", {});
   }
   {
     // Create a connection
-    Azure::Core::Amqp::_internal::Connection connection("amqps://localhost:5671", {});
+    Azure::Core::Amqp::_internal::ConnectionOptions connectOptions;
+    connectOptions.Port = Azure::Core::Amqp::_internal::AmqpPort;
+
+    Azure::Core::Amqp::_internal::Connection connection("localhost", connectOptions);
   }
   {
     Azure::Core::Amqp::_internal::ConnectionOptions options;
-    auto socketTransport{std::make_shared<Azure::Core::Amqp::Network::_internal::SocketTransport>(
-        "localhost", Azure::Core::Amqp::_detail::AmqpPort)};
+    auto socketTransport{Azure::Core::Amqp::Network::_internal::SocketTransportFactory::Create(
+        "localhost", Azure::Core::Amqp::_internal::AmqpPort)};
 
     Azure::Core::Amqp::_internal::Connection connection(socketTransport, options);
   }
-  {
-    Azure::Core::Amqp::_internal::ConnectionOptions options;
-    auto socketTransport{std::make_shared<Azure::Core::Amqp::Network::_internal::SocketTransport>(
-        "localhost", Azure::Core::Amqp::_detail::AmqpPort)};
-    options.SaslCredentials
-        = std::make_shared<Azure::Core::Amqp::_internal::SaslPlainConnectionStringCredential>(
-            "Endpoint=sb://testHost.net/"
-            ";SharedAccessKeyName=SomeName;SharedAccessKey=SomeKey;EntityPath=testhub");
-
-    EXPECT_ANY_THROW(Azure::Core::Amqp::_internal::Connection connection(socketTransport, options));
-  }
 
 #if 0
```

Comment:
a candidate to get uncommented or removed?

Reply:
Yeah - that was a github copilot authored bit of code that showed up REALLY early and I thought it looked cool :). Silly me.